CMakeLists.txt

```diff
@@ -101,12 +103,17 @@ install(FILES snappy.h
         ${Snappy_BINARY_DIR}/snappy-stubs-public.h
         DESTINATION include)
 
+if (BUILD_SHARED_LIBS)
```

Comment:
Wouldn't it be nicer to have both `BUILD_SHARED_LIBS` and `BUILD_STATIC_LIBS`, so you can build both a static and a shared library without reconfiguring in between?